topics/go/concurrency/goroutines/README.md

```diff
@@ -26,18 +26,18 @@ Goroutines are functions that are created and scheduled to be run independently
 
 ## Links
 
-http://blog.golang.org/advanced-go-concurrency-patterns  
-http://blog.golang.org/context  
-http://blog.golang.org/concurrency-is-not-parallelism  
-http://talks.golang.org/2013/distsys.slide  
+[Advanced Go Concurrency Patterns](https://blog.golang.org/advanced-go-concurrency-patterns) - Sameer Ajmani   
+[Go Concurrency Patterns: Context](https://blog.golang.org/context) - Sameer Ajmani   
+[Concurrency is not parallelism](https://blog.golang.org/concurrency-is-not-parallelism) - Rob Pike  
```

Comment:
All these lines need 2 extra spaces at the end so they CRLF.

Double check all links please.

Reply:
Sure, thanks for checking!
I've added four spaces to each link, PTAL.